scripts/controlnet.py

```diff
@@ -218,12 +220,15 @@ def uigroup(self, is_img2img):
         infotext_fields = []
         with gr.Row():
             input_image = gr.Image(source='upload', mirror_webcam=False, type='numpy', tool='sketch')
+            cropped_image = gr.Image(label="Crop image", interactive=True, tool="select", visible=False)
+        with gr.Row():
```

Comment:
This may break previous fix for https://github.com/Mikubill/sd-webui-controlnet/issues/311 - Is there any other way to make it smaller?

Reply:
I made the annotator result the same height as the input. I don't know if it can fit on the same row as cropped. Maybe I can make it fit with CSS.

Reply:
How about tabs? This way, you don't need the "hide annotator" button either.
![image](https://user-images.githubusercontent.com/4073789/222943692-c98a9e33-caaf-4b99-94d6-1d8056cbdf29.png)

And I was thinking "preview annotator" could be a toolbutton next to preprocessor. Something like 🔍.

Reply:
oh we are messing on buttons/icons. 
please check my pr #336 to see how i shrinked  buttons/rearranged. we would clash a bit here, if my pr gets merged

Reply:
I'll leave the buttona alone then.